docs/components/console/manage-clusters/manage-ip-allowlists.md

```diff
@@ -5,6 +5,10 @@ description: "If your organization works within Camunda's Enterprise plan, you c
 keywords: [whitelist, allowlist, ip whitelist, ip allowlist]
 ---
 
+:::note
+Connections from Web Modeler are automatically part of the allowlist.
```

Comment:
```suggestion
IP addresses required for Camunda SaaS functionality is automatically added to the allowlist. 
```

Reply:
Should we include any examples by name? Do users understand this separation of components? Similar to @multani's feedback. 

I'm good with either, but it may be something we want to monitor and adjust based on feedback.

Reply:
Could we phrase it in this way:
Camunda SaaS automatically manages Allowlist assignments for IP addresses used by Web Modeller.